app/liquid_tags/github_tag/github_issue_tag.rb

```diff
@@ -46,9 +53,19 @@ def parse_link(link)
     end
 
     def generate_api_link(input)
-      input = input.gsub(/\?.*/, "")
-      input = input.gsub(/\d{1,}#issuecomment-/, "comments/") if input.include?("#issuecomment-")
-      "https://api.github.com/repos/#{input.gsub(/.*github\.com\//, '')}"
+      path = input.gsub(/\?.*/, "")
+      path = path.gsub(/.*github\.com\//, "")
+
+      # issue fragments are ignored in the API call
+      path = path.gsub(/#issue-\d{1,}/, "") if path.include?("#issue-")
+
+      # GitHub's public PRs URLs are "/pull/{id}" but the API requires "/pulls/{id}"
+      path = path.gsub(/\/pull\//, "/pulls/") if path.include?("/pull/")
```

Comment:
this was missing, we ended up generating the wrong API url (we have a few of them in the DB)